event-consumer/manifest.yml

```diff
@@ -0,0 +1,35 @@
+packages:
+  __APP_PACKAGE__:
+    license: Apache-2.0
+    actions:
+      generic:
+        function: actions/generic/index.js
+        web: 'yes'
+        runtime: 'nodejs:12'
+        inputs:
```

Comment:
looks like it's missing the sample dot-env file?

Reply:
added one